src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -19,42 +18,59 @@ The following options can be configured on the root level of your browser-based
-| stickySession | boolean | `true`  | Keep track of the user across page loads. Note, a single user using multiple tabs will result in multiple sessions. Closing a tab will result in the session being closed as well. |
+| Key                    | Type                 | Default | Description                                                                                                                                                   |
+| ---------------------- | -------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| stickySession          | boolean              | `true`  | Keep track of users across page loads. Note, because closing a tab ends the session, a single user using multiple tabs will be recorded as multiple sessions. |
+| networkDetailAllowUrls | (string \| RegExp)[] | `[]`    | Capture request and response details for XHR and Fetch requests that match the given URLs.                                                                    |
+| networkCaptureBodies   | boolean              | `true`  | Decide whether to capture request and response bodies for URLs defined in `networkDetailAllowUrls`.                                                           |
+| networkRequestHeaders  | string[]             | `[]`    | Capture request headers for URLs defined in `networkDetailAllowUrls`. See [Network Details](#network-details) to learn more.                                                                               |
+| networkResponseHeaders | string[]             | `[]`    | Capture response headers for URLs defined in `networkDetailAllowUrls`. See [Network Details](#network-details) to learn more.                                                                              |
 
-## Identifying Users
+### Sampling
+
+If you want to understand how exactly our session sampling works, check out our <PlatformLink to="/session-replay/understanding-sessions/#default-session-initialization">Understanding Sessions > Default Session Initialization</PlatformLink> documentation.
+
+### Network Details
+
+By default, Replay will capture basic information about all outgoing fetch and XHR requests in your application. This includes the URL, request and response body size, method, and status code. The intention is to limit the chance of collecting private data.
 
-You can use the Sentry SDK to set the user of a session. To associate a user identity to a session replay, refer to the <PlatformLink to="/enriching-events/identify-user">Identifying User Docs</PlatformLink>.
+To capture additional information such as request and response headers or bodies, you'll need to opt-in via `networkDetailAllowUrls`. This gives you control to add only URLs that are safe to capture bodies, avoiding any endpoints that can contain personal identifiable information (i.e., PII).
+
```

Comment:
```suggestion
To capture additional information such as request and response headers or bodies, you'll need to opt-in via `networkDetailAllowUrls`. This will make it possible for you to elect to only add URLs that are safe for capturing bodies and avoid any endpoints that may contain Personal Identifiable Information, (PII).

```